modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,62 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a configuration for assignment of dualstack IP addresses dynamically
+
+DualStack IP address assignment can be configured with the `ipRanges` parameter for:
```

Comment:
nit: Let's change to "dual-stack" with a hyphen. On this line and the one just above it. Looks like it's the most common in the docs. Here's my count of the variations in the docs:

```
$ grep -Prin "dualstack" * | wc -l
20
$ grep -Prin "dual.stack" * | wc -l
124
$ grep -Prin "dual stack" * | wc -l
24
$ grep -Prin "dual\-stack" * | wc -l
100
```

Reply:
Sure thing